packages/postgres-database/src/simcore_postgres_database/models/projects.py

```diff
@@ -16,6 +16,12 @@ class ProjectType(enum.Enum):
     STANDARD = "STANDARD"
 
 
+class ProjectTemplateType(str, enum.Enum):
+    TEMPLATE = "TEMPLATE"
```

Comment:
A project template type is template. It sound strange, right? What about BLUEPRINT or CLONEABLE or ARCHETYPE or STARTER?